pandas/core/arrays/base.py

```diff
@@ -88,14 +88,16 @@ class ExtensionArray(object):
     # Constructors
     # ------------------------------------------------------------------------
     @classmethod
-    def _from_sequence(cls, scalars, copy=False):
+    def _from_sequence(cls, scalars, dtype=None, copy=False):
         """Construct a new ExtensionArray from a sequence of scalars.
 
         Parameters
         ----------
         scalars : Sequence
             Each element will be an instance of the scalar type for this
             array, ``cls.dtype.type``.
+        dtype : Dtype, optional
+            consruct for this particular dtype
```

Comment:
This needs to be a dtype compatible with the ExtensionArray?

Reply:
yes

Reply:
Can you clarify that in the docstring? 
Because basically what it comes down to is to be able to pass parameters of the dtype (in this case the actual dtype (int8, int64, ..), or in case of the Categorical the categories/ordered attributes, to have a dtype faithfully round-trip

Reply:
done